etl/xml_importer/entities/artwork.py

```diff
@@ -48,20 +49,35 @@ def __init__(self, lido):
 
         self.clear()
 
+    # TODO: diese Attribute haben eine niedrige Prio, daher erstmal nicht weiter beachten
+    # artwork.altName string[] 2
+    # artwork.inscriptions string[] 2
+
     def _parse_id(self):
+        # TODO: Was soll ich hier nur lido herausnehmen?: DE - Mb112 - lido - t3 - 000230
+        #         88 - T - 001 - T - 065
         id = self.lido.find(paths["Artwork_Id_Path"], namespace).text
         id = sanitize_id(id)
         id = id.replace("/", "-").replace(",", "-").replace("lido-", "").replace("obj-", "")
 
         self.id = self.entity_type + "-" + id
 
-    def _parse_label(self):
+    def _parse_label(self):  # TODO: Format DE-Mb112-00000000001
         label = self.lido.find(paths["Artwork_Name_Path"], namespace)
         if label is not None:
             self.label = sanitize(label.text)
         else:
             self.label = None
 
+    def _parse_alt_label(self):
+        self.alt_labels = []
+        altLabels_list = self.lido.findall(paths["Artwork_AltLabel_Path"], namespace)
+        if altLabels_list is not None:
```

Comment:
This check is not necessary here since findall(...) returns an empty list if nothing was found, which can be safely iterated. Please rename them to altLabel_roots and `element` to `root` or `altLabel_root` to keep it consistent.
```suggestion
       altLabel_roots = self.lido.findall(paths["Artwork_AltLabel_Path"], namespace)
       for root in altLabel_roots:
                self.alt_labels.append(sanitize(root.text))
```